echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/NotificationTemplateEngine.groovy

```diff
@@ -45,12 +45,12 @@ class NotificationTemplateEngine {
     String build(Notification notification, Type type) {
         if (!notification.templateGroup) {
           if (type == Type.SUBJECT) {
-            return (FORMATTERS.find { it.type == notification.additionalContext.formatter }?: new MarkdownToHtmlFormatter())
+            return (FORMATTERS.find { it.type == notification.additionalContext.formatter }?: new MarkdownPassThruFormatter())
```

Comment:
I believe this was a bug caused by recent updates in the way Slack renders text/markdown/HTML. Instead of rendering formatted content, Slack was displaying the HTML code. Having a pass-through formatter allows markdown to render correctly.